### PR TITLE
Fix unvisited JSON members

### PR DESCRIPTION
**Pull request by anothersimulacrum**
#### Summary
None

#### Purpose of change
Working towards re-enabling unvisited JSON member checking (see https://github.com/CleverRaven/Cataclysm-DDA/issues/64438).
Fix JSON errors.

#### Describe the solution
Enable unvisited member checking, game complains about these.

#### Additional context
Mods still need to be done.
Alerting the following people in case these data errors need other solutions
@dseguin @Venera3 https://github.com/CleverRaven/Cataclysm-DDA/commit/48b36faec137562960964170727325463a99c004
@MNG-cataclysm @LovamKicsiGazsii https://github.com/CleverRaven/Cataclysm-DDA/commit/a773ecec5e14e5d5419a68574ee70ebdf4af95a5 https://github.com/CleverRaven/Cataclysm-DDA/commit/70dc95c06db3e4659c1452609f8b25b362d15a39 https://github.com/CleverRaven/Cataclysm-DDA/commit/3aca0a2fd56ac5e6fc1c2a39d2247960b24edb14
@GuardianDll https://github.com/CleverRaven/Cataclysm-DDA/commit/8ad015a6af7073b1c0960bd46056cc690652ab6b https://github.com/CleverRaven/Cataclysm-DDA/commit/673a65cc15a0d10722ca0b0f5ac1e3b0a84467d8

### CODE REVIEW

**Review by Venera3 — APPROVED** [review]
Dodge encumb got scored properly, some dumdum forgot to remove the field. Thanks!

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -7959,7 +7959,6 @@
     "points": 99,
     "valid": false,
     "description": "If you need to survive the deadliest bug attacks.",
```
Comment (GuardianDll) [inline data/json/mutations/mutations.json:7961]:
```suggestion
    "description": "If you need to survive the deadliest bug attacks.",
    "enchantments": [ { "values": [ { "value": "MAX_HP", "add": 9000 } ] } ],
```

File: data/json/mutations/mutations.json [hunk 2]
```diff
@@ -7969,7 +7968,6 @@
     "points": 99,
     "valid": false,
     "description": "Heal yourself from this stupid bug attacks.",
```
Comment (GuardianDll) [inline data/json/mutations/mutations.json:7970]:
```suggestion
    "description": "Heal yourself from this stupid bug attacks.",
    "enchantments": [ { "values": [ { "value": "REGEN_HP", "add": 100 } ] } ],
```

**Comment by anothersimulacrum** [comment]
It will work with no `to` field defined, but it expects a string for that field if it is defined. I'll remove the field.

**Review by FarkasKBence — APPROVED** [review]
At first I was confused why you removed the `repeat`s from the private resort, but then it dawned on me that distribution doesn't work like that.🤦

**Review by GuardianDll — APPROVED** [review]
Kinda sad the variants do not accept variety in stats, but thats okay

**Pull request merged**